lib/ngx/ssl.md

```diff
@@ -299,6 +300,43 @@ This function can be called in whatever contexts where downstream https is used.
 
 [Back to TOC](#table-of-contents)
 
+client_addr
+---------------
+**syntax:** *addr, err = ssl.client_addr()*
+
+**context:** *any*
+
+Returns the client ip address in the current SSL connection.
+
+The following code snippet shows how to delaying TLS handshake to mitigate a kind of TSL handshake attack(eg: close connection immediately after handshake complete):
```

Comment:
I am not a native English speaker, so please correct me if I am wrong. I think it should be `how to delay` instead of `how to delaying`? And `after handshake complete` should be `after handshake completed`?

Moreover, it should be `TLS` instead of `TSL`.